doc/REST-interface.md

```diff
@@ -36,7 +36,7 @@ By default, this endpoint will only search the mempool.
 To query for a confirmed transaction, enable the transaction index via "txindex=1" command line / configuration option.
 
 #### Blocks
-`GET /rest/block/<BLOCK-HASH>.<bin|hex|json>`
+`GET /rest/block/<BLOCK-HASH>.<bin|hex|json>` <br>
```

Comment:
Markdown is supposed to be readable as plain text.

Make this a list instead?

Reply:
Right, this also explains why there is no other instance of `<br>` in any of our Markdown docs. Changed it to a list (here an for the blocks endpoint), as you suggested.